src/unix/notbsd/linux/other/mod.rs

```diff
@@ -689,12 +689,7 @@ pub const NFPROTO_NETDEV: ::c_int = 5;
 
 // linux/netfilter/nf_tables.h
 cfg_if!{
-    if #[cfg(any(target_arch = "arm", target_arch = "aarch64"))] {
```

Comment:
Sparc64 was so different because it had ancient kernel, if 19.04 build succeeds this will cause an error.

Reply:
I'll adapt this if the 19.04 succeeds, but it appears that 19.04 fails for other reasons everywhere.

Reply:
It changed values all over the place (quite expected) and sparc64 executes on old Debian with very old glibc: https://github.com/rust-lang/libc/blob/master/ci/linux-sparc64.sh#L8

Reply:
Ah, the sparc64 tests run on qemu-system using the disk image in that file. In that case, changing the Ubuntu version of the Docker container should have any effect right? What matters is the glibc version of the image used within qemu-system.

Reply:
There is this failure:
```
cargo:warning=/checkout/target/sparc64-unknown-linux-gnu/debug/build/libc-test-bf57c51a205470a0/out/main.c:21088:28: error: returning '__int32_t *' {aka 'int *'} from a function with incompatible return type 'long int *' [-Werror=incompatible-pointer-types]
cargo:warning=                     return &b->ut_session;
cargo:warning=                            ^~~~~~~~~~~~~~
cargo:warning=/checkout/target/sparc64-unknown-linux-gnu/debug/build/libc-test-bf57c51a205470a0/out/main.c: In function '__sparc_get_pc_thunk.l7':
cargo:warning=cc1: error: unrecognized command line option '-Wno-address-of-packed-member' [-Werror]
cargo:warning=cc1: all warnings being treated as errors
```
And if you build executable on Ubuntu with glibc 2.29 it won't run on Debian with much older glibc.

Reply:
Is that so? I was pretty certain that glibc is dynamically linked, so as long as you only use features of glibc 2.29 that are also available in older glibc versions that should work fine.

Reply:
The thing is it tries to pick some 2.27 feature, latest build:
```
/prog: /lib/libc.so.6: version `GLIBC_2.27' not found (required by /prog)
```

Reply:
Debian 10 has glibc 2.28, it could possibly work for now.

Reply:
So I've updated the debian images used in the sparc64 and s390x build jobs, we'll see what happens.